ReactiveCocoa/UIKit/UIGestureRecognizer.swift

```diff
@@ -0,0 +1,25 @@
+import ReactiveSwift
+import UIKit
+import enum Result.NoError
+
+extension Reactive where Base: UIGestureRecognizer {
+	
+	/// Create a signal which sends a `next` event for each gesture event
+	///
+	/// - returns:
+	///   A trigger signal.
+	
+	public var recognised: Signal<(), NoError> {
```

Comment:
I would pass the `UIGestureRecognizer` (so `Base`). Users normally need to look at at least `UIGestureRecognizer.state`, for also things like `location`, etc.